public/app/percona/settings/components/Platform/Platform.messages.ts

```diff
@@ -5,4 +5,5 @@ export const Messages = {
   pmmServerId: 'PMM Server Id',
   accessToken: 'Percona Platform Access Token',
   connect: 'Connect',
+  invalidToken: 'The token is expired, please update the token',
```

Comment:
Can we rephrase it, maybe "please update it"?

Reply:
Good suggestion, done.